### PR TITLE
Updates to ESLint 2.0.0

### LINKED ISSUE #79: Bump peerDependencies to support ESLint 2.0.0
[2.0.0](http://eslint.org/blog/2016/02/eslint-v2.0.0-released) was just released. Good time to bump. :+1:

### PR DESCRIPTION
**Pull request by farneman**
Updates ESLint dependencies to 2.0.0 and fixes some related test failures. 

This fixes #79

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,14 +16,14 @@
     "index.js"
   ],
   "peerDependencies": {
-    "eslint": "^1.6.0"
+    "eslint": "^2.0.0"
```
Comment (MoOx) [inline package.json]: Since there is no major change in the API, maybe we can scope 1.6 AND 2.0 ?

File: test/eslintignore.js [hunk 2]
```diff
@@ -8,11 +8,7 @@ test("eslint-loader ignores files present in .eslintignore", function(t) {
     conf,
     {
       entry: "./test/fixtures/ignore.js",
-      eslint: assign({}, conf.eslint, {
-        // we want to enable ignore, so eslint will parse .eslintignore and
-        // should skip the file specified above
-        ignore: true,
-      }),
+      eslint: assign({}, conf.eslint),
```
Comment (MoOx) [inline test/eslintignore.js]: why is this change required?

**Comment by farneman** [comment]
I updated the version specification and took another look at the test. It turns out I misunderstood what was going on in the test. However, without change that test fails so I need to sort out why and fix it correctly.

**Comment by farneman** [comment]
All tests are passing now. The skip ignored file warning conditional had to be updated due to the change at https://github.com/eslint/eslint/commit/d43f26c270f0070ca47000690d874d126d67fd89#diff-3daec82d5218d64d8f2fed4d21f8e6ccR291.

File: index.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ function lint(input, config, webpack) {
     res.warningCount === 1 &&
     res.results[0].messages[0] &&
     res.results[0].messages[0].message &&
-    res.results[0].messages[0].message.indexOf(".eslintignore") > -1 &&
+    res.results[0].messages[0].message.indexOf("ignore") > -1 &&
```
Comment (MoOx) [inline index.js]: if you try "ignore", no need for "no-ignore"?

**Comment by farneman** [comment]
I've reduced the to string checks to just one check for more than one occurrence of `ignore`.

File: package.json [hunk 4]
```diff
@@ -16,14 +16,14 @@
     "index.js"
   ],
   "peerDependencies": {
-    "eslint": "^1.6.0"
+    "eslint": ">=1.6.0 <2.1.0"
```
Comment (MoOx) [inline package.json]: If api respect semver, should be <3.0.0 ?

**Comment by farneman** [comment]
I updated the ESLint dependency scope.

**Comment by omichelsen** [comment]
:+1:

**Comment by alyssaq** [comment]
:+1: Waiting on this patch to merge to upgrade!

**Comment by MoOx** [comment]
Thanks!

**Comment by MoOx** [comment]
Released as 1.3.0
